CMSIS/Core/Include/cmsis_iccarm.h

```diff
@@ -961,4 +961,6 @@ __packed struct  __iar_u32 { uint32_t v; };
 #pragma diag_default=Pe940
 #pragma diag_default=Pe177
 
+#define __SXTB16_RORn(ARG1, ARG2) __SXTB16(__ROR(ARG1, ARG2))
```

Comment:
@TTornblom, may I ask you to approve this enhancement to your compiler header, please?

- [ ] Approved by IAR @TTornblom